Add subpages to the menubar

svirvel opened the pull request:
Auktionssidan behöver ha flera undersidor för att t ex bläddra i olika delar av katalogen. Den nuvarande lösningen är att ha en sidomenu på alla auktionssidor. En snyggar och mera effektiv lösning är att lägga till stöd för undersidor till huvudmenyn. Mitt förslag är att lägga till undersidor till huvudmenyn som "dropdowns", men att lägga dessa som en statisk horisontell meny under huvudmenyn som visas på t ex alla auktionssidor.

Den föreslagna undermenyn har i brist av färgschema och för att hedra ETAs ursprung en gul bakgrundsfärg. Färgschema, design och funktionalitet av menyn behöver diskuteras vidare.

svirvel committed:
Add subnavigation
Add subnavigation by using bootstraps dropdown menu to list child pages
for menues. Modifies the css for the dropdown menu when the main
navigation bar is in it's expanded state to create a horizontal submenu
below the main navigation.

svirvel committed:
Add auction pages to main menu
Include all auctionpages to the main menu. Set the main auciton page as
parent for all item listing pages. Remove the sidebar on the
auctionpages.

almbr commented:
**Strukturen av hemsidan:**
Personligen tycker jag att den lösning du nu har utvecklat är något förvirrande. Om auktionshemsidan ska ha dropdowns tycker jag att de ska vara: Information, Auktionsgods. 

Jag tror att det blir förvirrande för besökaren att så tidigt i "hemsideprocessen" få välja gods. Mer logiskt tykcer jag vore att först presentera denna undermeny efter att auktionshemsidan besökts. Eller till och med att "Auktionsgods"-hemsidan besökts.

**Färg:**
Gällande den för ETA anrika färgen gul kan jag inte säga emot. Dock tycker jag att det passar bättre att denna används för detaljer på hemsidan såsom mindre punkter, liner, mm... Den navbar du byggt tror jag hade passat bäst i svart.

svirvel committed:
Sort main menu entries by weight
Add weight to the home page in the main menu to put "Home" to the left
in the menubar.

svirvel commented:
**Struktur:**
Håller med dig @tsuum att det kan verka förvirrande, däremot visas undermenyn först när auktionssidan är besökt. Tanken är att det ska finnas information om auktionen på /auction. Därför känns det onödigt att ha information som en separat sida. Känns då som att det blir konstigt att ha en ensam dropdown för auktionsgods som ytterligare har en meny. Tror även att det är bra att referera till katalogen tidig för att det ska gå enkelt att hitta allt.

Det hade gått att ha en fullständig auktionskatalog på /auction och skippa separata sidor, men tror att det blir svårare att på att bra sätt navigera i katalogen då. 

Har du något annat förslag på hur det hade kunnat göras? @tsuum 

**Färg:**
Jag tycker att svart ger lite för stor kontrast mot resterande delar av sidan och att det då ser konstigt ut. Jag uppdaterade den istället till att använda bootstraps gråa bakgrundsfärg vilket ser mer neutral ut. Hur tycker du att det ser ut? För att göra det enklare med designen bör ett färgtema skapas #12 .

svirvel commented:
Gjorde en ytterligare ändring av huvudmenyn för att ordningen på de olika alternativen ska vara mer logisk.
